index.html

```diff
@@ -670,9 +684,10 @@ <h2>Profiles</h2>
         </p>
 
         <pre class="example nohighlight" title="Basic structure of a profile">
-{
-  "id": "did:example:ebfeb1f712ebc6f1c276e12ec21",
-  "type": ["VerifiableProfile"],
+{"verifiableProfile": {
+  "id": "< profile id >",
```

Comment:
This is not guaranteed to be there, nor does it need to be.

Reply:
I was proposing that the ID becomes mandatory.

Reply:
We should point to use cases where an ID is not desirable. Or we could add a note about how the ID may be omitted to support certain privacy requirements.